provider/schema/resource.go

```diff
@@ -32,6 +34,10 @@ func (r Resource) Get(key string) interface{} {
 }
 
 func (r Resource) Set(key string, value interface{}) {
+	columnExists := funk.ContainsString(r.table.ColumnNames(), key)
+	if !columnExists {
+		panic(fmt.Sprintf("column %s does not exits", key))
```

Comment:
Exists*, and maybe it is better to return an error here instead (I know it will basically means we have to update all providers to check / return error), it is better than a panic imo. Providers can update to this new syntax later.